packages/nightingale-navigation/src/nightingale-navigation.ts

```diff
@@ -91,16 +88,16 @@ class NightingaleNavigation extends withManager(
       ])
       .on("brush", ({ selection, transform }) => {
         if (selection && this.#x) {
-          this['display-start'] =
+          this["display-start"] =
```

Comment:
Sorry, I didn't notice my prettier is setup differently. We should add a prettier config to the project to make sure we avoid this sort of changes.